src/TextEditor.js

```diff
@@ -149,11 +149,26 @@ export default class TextEditor extends Component {
    * Keyboard shortcuts
    */
   handleKeyCommand(command) {
-    const newEditorStatue = RichUtils.handleKeyCommand(this.state.editorState, command);
-    if (newEditorStatue) {
+    let content;
+    let editor;
+    let {editorState} = this.state;
+
+    const newEditorStatue = RichUtils.handleKeyCommand(editorState, command);
+
+    // Split the selected block into two blocks on 'Enter' command.
+    // @see https://draftjs.org/docs/api-reference-modifier.html#splitblock
+    if (command === 'split-block') {
```

Comment:
We should probably only do this in `html` mode. It'll probably cause issues with the other modes